src/app/account-app/runbox-timer.ts

```diff
@@ -0,0 +1,137 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2018 Runbox Solutions AS (runbox.com).
```

Comment:
Should be 2020 now I suppose :) I'm no expert on legal matters, but should be strive to keep this up-to-date? Perhaps a test (with an updater) should be introduced, that makes sure our copyrights are up-to-date?